src/AbstractFixer.php

```diff
@@ -166,7 +166,7 @@ public function getConfigurationDefinition()
     public function setWhitespacesConfig(WhitespacesFixerConfig $config)
     {
         if (!$this instanceof WhitespacesAwareFixerInterface) {
-            throw new \LogicException('Cannot run method for class not implementing `WhitespacesAwareFixerInterface`.');
+            throw new \LogicException('Cannot run method for class not implementing "PhpCsFixer\Fixer\WhitespacesAwareFixerInterface".');
```

Comment:
why removing backticks ?

Reply:
these are replaced to be the same as done @ line 156